src/App.js

```diff
@@ -53,6 +111,16 @@ class App extends Component {
         <option key={this.state.technologies[i].name} value={this.state.technologies[i].name} />
       )
     }
+
+    sortOrder = <SortOrderTechAsc onClick={this.handleSort} />;
+    if(this.state.sortBy !== 'tech' && !this.state.orderDesc){
+      sortOrder = <SortOrderAgeAsc onClick={this.handleSort} />;
+    }else if(this.state.sortBy !== 'tech' && this.state.orderDesc){
+      sortOrder = <SortOrderAgeDesc onClick={this.handleSort} />;
+    }else if(this.state.sortBy === 'tech' && this.state.orderDesc){
+      sortOrder = <SortOrderTechDesc onClick={this.handleSort} />;
+    }
+
```

Comment:
Rather than writing all these functions use conditionals, something like this
```
<div className="inline">
  <p className="inline">Order by</p>
  <div
    className={this.state.orderBy === "tech" ? "inline bold" : "inline"}
    onClick={props.onClick}
  >
    Tech
  </div>
  {this.state.orderBy === "tech" && (
    <div className={this.state.orderDesc ? "arrow-down" : "arrow-up"} />
  )}
  <div
    className={this.state.orderBy !== "tech" ? "inline bold" : "inline"}
    onClick={props.onClick}
  >
    Age
  </div>
  {this.state.orderBy !== "tech" && (
    <div className={this.state.orderDesc ? "arrow-down" : "arrow-up"} />
  )}
</div>
```